modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java

```diff
@@ -1460,6 +1511,68 @@ private void _addObjectRelationshipERCFieldValue(
 		}
 	}
 
+	private String _createInsertIntoLocalizationTableStatement(
+			DynamicObjectDefinitionLocalizationTable
+				dynamicObjectDefinitionLocalizationTable,
+			Map<String, Serializable> values)
+		throws PortalException {
+
+		if (dynamicObjectDefinitionLocalizationTable == null) {
+			return null;
+		}
+
+		List<ObjectField> objectFields =
+			dynamicObjectDefinitionLocalizationTable.getObjectFields();
+
+		if (objectFields.isEmpty()) {
+			return null;
+		}
+
+		StringBundler sb = new StringBundler();
+
+		sb.append("insert into ");
+		sb.append(dynamicObjectDefinitionLocalizationTable.getName());
+		sb.append(" (");
+		sb.append(
+			dynamicObjectDefinitionLocalizationTable.getPrimaryKeyColumnName());
+		sb.append(", ");
+		sb.append(
+			dynamicObjectDefinitionLocalizationTable.getForeignKeyColumnName());
+		sb.append(", languageId");
+
+		int columnsCount = 3;
```

Comment:
rename to `count`

Reply:
Done [here](https://github.com/liferay-objects/liferay-portal/pull/1913/commits/72641e031065c63619c3dceccebf57c9604cf77e)